vpr/src/route/route_timing.h

```diff
@@ -13,6 +13,8 @@
 #include "connection_router_interface.h"
 #include "heap_type.h"
 
+extern bool f_router_debug;
```

Comment:
I had missed this comment.
So, f_router_debug gets used in the extended lookahead map code to enable debugging prints of the returned lookahead costs (with all the various details such as delay, congestion, site_pin_delay, etc.).

Usure how to bring it in the lookahead if it gets commented here. I need to see if there is another solution.

Reply:
This may need a new variable then.  Adding @MohamedElgammal  to comment.  If f_router_debug means "stopped at a breakpoint" (we probably need to rename the variable to f_router_breakpoint in that case) then we shouldn't use it to control whether or not we print out additional lookahead info.  Should be a different variable then.

Reply:
Please update the comment in route_timing.cpp to explain what this variable does.  The comment currently says it is for breakpoints, which is no longer true (which is good news).  Explain it is to output extra debugging info.